Tryouts/Prototypes/Shell/Node-launcher/Lib/rollup.config.js

```diff
@@ -0,0 +1,13 @@
+﻿import { nodeResolve } from '@rollup/plugin-node-resolve';
+import typescript from '@rollup/plugin-typescript';
+
+export default {
+    input: 'src/index.ts',
+    output: {
+      
+        file: 'output/bundle.mjs',
+        format: 'es',
+        name: "@compose/node-launcher"
```

Comment:
Why is the scope and name different from the one in `package.json`? Is this something rollup-specific?

Reply:
It's the global variable name that represents the bundle if it's iife or umd, But it's not needed for ES so I'm going to remove it.